src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -57,6 +59,26 @@ public class JsonWebToken : SecurityToken
         private string _x5t;
         private string _zip;
 
+        /// <summary>
+        /// Gets or sets the property name of <see cref="Claim.Properties"/> the will contain the original JSON claim 'name' if a mapping occurred when the <see cref="Claim"/>(s) were created.
+        /// </summary>
+        /// <exception cref="ArgumentException">If <see cref="string"/>.IsNullOrWhiteSpace('value') is true.</exception>
+        public static string ShortClaimTypeProperty
```

Comment:
We should have one of these properties in JwtSecurityTokenHandler and JsonWebTokenHandler having a static shared between the two mean that changing JsonWebToken.ShortClaimTypeProperty will affect JwtSecurityToken, that is not desirable.

Reply:
Each handler will have their own version of this property